pysite/views/wiki/edit.py

```diff
@@ -24,16 +30,31 @@ def get(self, page):
         obj = self.db.get(self.table_name, page)
 
         if obj:
-            rst = obj["rst"]
-            title = obj["title"]
-            preview = obj["html"]
+            rst = obj.get("rst", "")
+            title = obj.get("title", "")
+            preview = obj.get("html", preview)
+
+            if obj.get("lock_expiry") and obj.get("lock_user") != self.user_data.get("user_id"):
+                lock_time = datetime.datetime.fromtimestamp(obj["lock_expiry"])
+                if datetime.datetime.utcnow() < lock_time:
+                    return self.render("wiki/page_in_use.html", page=page)
+
+        lock_expiry = datetime.datetime.utcnow() + datetime.timedelta(minutes=5)
+
+        if not DEBUG_MODE:  # If we are in debug mode we have no user logged in, therefore we can skip locking
+            self.db.insert(self.table_name, {
```

Comment:
Not a fan of this line continuation - can't you just start on the next line with a 4 space indent instead?